### PR TITLE
Move platform_channel example to latest template

### PR DESCRIPTION
**Pull request by cbracken**

### CODE REVIEW

**Comment by cbracken** [comment]
Fixes #8062 for platform_channel

File: examples/platform_channel/ios/Runner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -119,18 +122,26 @@
 			name = "Supporting Files";
 			sourceTree = "<group>";
 		};
+		CF3B75C9A7D2FA2A4C99F110 /* Frameworks */ = {
```
Comment (xster) [inline examples/platform_channel/ios/Runner.xcodeproj/project.pbxproj]: Is this intentional? Suspect we might not need this

**Review by xster — APPROVED** [review]
LG

File: examples/platform_channel/ios/Runner.xcodeproj/project.pbxproj (hunk 1 above)
Comment (cbracken) [inline examples/platform_channel/ios/Runner.xcodeproj/project.pbxproj]: Yeah - this is in keeping with the others. While it starts empty, it's also not an uncommon setup. We just as well could kill it off everywhere too.

**Pull request merged**